src/test/java/uk/gov/companieshouse/personapi/controller/PersonControllerTest.java

```diff
@@ -69,7 +77,37 @@ void testGetPersonByIdThrowsPersonNotFoundException() throws Exception {
                 .andExpect(jsonPath("$.timestamp",any(String.class)))
                 .andExpect(jsonPath("$.message",is(ERROR_MESSAGE)));
 
+    }
+
+    @Test
+    void testGetPeopleSuccessful() throws Exception {
+        Person person1 = (Person) people.get(0);
+        Person person2 = (Person) people.get(1);
+        List<Person> people1 = new ArrayList<>();
+        people1.add(person1);
+        people1.add(person2);
+        when(personService.getPeople()).thenReturn(people1);
+
+        mvc.perform(get("/people"))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$[0].forename", is(person.getForename())))
+                .andExpect(jsonPath("$[1].forename", is(obj.getForename())));
+    }
 
+    @Test
+    void testDeletePersonReturnsOkWhenSuccessful() throws Exception {
+        doNothing().when(personService).deletePerson(PERSON_ID); // assumes successful deletion as no exceptions thrown
+        mvc.perform(delete("/delet-person/" + PERSON_ID))
+                .andExpect(status().isOk());
     }
 
+    @Test
+    void test() throws Exception {
+        doThrow(new Exception("error")).when(personService).deletePerson(PERSON_ID);
```

Comment:
code is inconsistent with previous code, maybe either change this or previous to be the same.

Reply:
The reason it was done this way is due to the `deletePerson(PERSON_ID)` method being void. Using `when(personService.deletePerson(PERSON_ID))` will not work for void methods, however `doThrow(new Exception("")).when(personService).deletePerson(PERSON_ID)` works for both void and returned methods.
I have changed them to both use the `doThrow()` way.